### DISCUSSION TITLE
The need to translate the names of countries into each language

### DISCUSSION THREAD

**Discussion by andrey-helldar** [Laravel-Lang/common#46] [Q&A]
We recently spoke with @fcolecumberri about the [need to translate](https://github.com/Laravel-Lang/lang/pull/1566#discussion_r601256520) the list of countries into each language.
@fcolecumberri said that such a translation was needed, to which I replied that Russian-language sites rarely translate countries into Russian and we often use names in English localization.

And it is true. Outdated truth. I talked with people and we came to the conclusion that really before the list of countries was not required for translation due to human laziness - people were too lazy to translate. But since about 2010, websites with the translation of country names began to appear.

So @fcolecumberri, I was wrong. PR with translation of country names: https://github.com/Laravel-Lang/lang/pull/1582, https://github.com/Laravel-Lang/lang/pull/1583

@caouecs, you ask why this discussion then? The answer is: how do you look at removing the list of countries from the ignore list, thus requiring the need for their translation?

**Comment by caouecs** [comment]
I don't know the number of languages which don't translate country names. I was surprise for Russian.

For me, all languages need to translate country names, but we can do a post with juste the list of countries, if a language doesn't translate, we can't copy/paste the list in exclude file

**Reply by andrey-helldar** [reply]
Yes, we used to have a list of countries in English. Now people are trying to translate from English.
In this case, if someone has the same name as English, then they will have to specify these keys in the exclude file (`excludes/<locale>.php`).

**Comment by fcolecumberri** [comment]
Making a fast duckduckgo search, I found these pages https://en.wikipedia.org/wiki/List_of_country_names_in_various_languages_(A%E2%80%93C) https://en.wikipedia.org/wiki/List_of_country_names_in_various_languages_(D%E2%80%93I) https://en.wikipedia.org/wiki/List_of_country_names_in_various_languages_(J%E2%80%93P) https://en.wikipedia.org/wiki/List_of_country_names_in_various_languages_(Q%E2%80%93Z) maybe it could be easier to just make a script to extract that information, hope that wikipedia is correct and if something is wrong, people will make the pull request.

**Reply by andrey-helldar** [reply]
It is good when it is possible to correctly translate the names of the countries into the desired language.

But it's hard for one or two people to do it, because there is too much work.

That is why we have a team in which each of its members is [assigned](https://github.com/Laravel-Lang/lang/blob/master/REFERENTS.md) to the chosen localization.
